pfrl/agents/goal_conditioned_td3.py

```diff
@@ -265,6 +267,23 @@ def update_policy_with_goal(self, batch):
         self.policy_optimizer.step()
         self.policy_n_updates += 1
 
+        action_distrib = self.policy(torch.cat([batch_state, batch_goal], -1))
+        target_action_distrib = self.target_policy(torch.cat([batch_state, batch_goal], -1))
```

Comment:
didn't you want to compare to the policy from the previous iteration, not the target policy?

Reply:
We can do both. The reason why I chose the target policy for comparison is because it's more stable than the the policy that is updated every time, meaning that the KL divergence will be quite large for both deterministic and stochastic policies. Instead, if we evaluate current policy against the policy from a while back, I think the difference between deterministic case and stochastic case would be evident. However, we can try both see how it turns out.

Reply:
Yea, that would be great.

Reply:
fixed